lib/src/parser.c

```diff
@@ -185,7 +185,7 @@ static bool ts_parser__breakdown_top_of_stack(
       TSStateId state = ts_stack_state(self->stack, slice.version);
       Subtree parent = *array_front(&slice.subtrees);
 
-      for (uint32_t j = 0, n = ts_subtree_child_count(parent); j < n; j++) {
+      for (uint32_t j = 0, cnt = ts_subtree_child_count(parent); j < cnt; j++) {
```

Comment:
What is the rationale for some of these renames? `n` seems pretty standard for a count. `cnt` is a bit weird IMO. If we need an abbreviated word I'd go w/ `len`.

Reply:
Honestly it's because clang-tidy suggests to not use variables less than 3 chars long, but for a loop variable (like i, j, k) it's accepted, might be a bug where multiple loop variables aren't picked up. I'd agree n is fine here